### PR TITLE
Add a note about the agent status table limit of 10k records

### PR DESCRIPTION
**Pull request by kilfoyle**
The updates [View agent status overview](https://www.elastic.co/guide/en/fleet/current/monitor-elastic-agent.html#view-agent-status) to note that the table is limited to 10k records.

Closes: #296

---

Screenshot 2024-11-04 at 12 43 05 PM

---

This also adds a note about the table sorting:

---

Screenshot 2024-11-05 at 9 34 44 AM

### CODE REVIEW

File: docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc [hunk 1]
```diff
@@ -33,6 +33,8 @@ To view the overall status of your {fleet}-managed agents, in {kib}, go to
 [role="screenshot"]
 image::images/kibana-fleet-agents.png[Agents tab showing status of each {agent}]
 
+IMPORTANT: The **Agents** tab in **{fleet}** displays a maximum of 10,000 agents, shown on 500 pages with 20 rows per page. If you have more than 10,000 agents, we recommend using the filtering and sorting options described in this section to narrow the table to fewer than 10,000 rows.
```
Comment (kilfoyle) [inline docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc:36]:
```suggestion
IMPORTANT: The **Agents** tab in **{fleet}** displays a maximum of 10,000 agents, shown on 500 pages with 20 rows per page.
If you have more than 10,000 agents, we recommend using the filtering and sorting options described in this section to narrow the table to fewer than 10,000 rows.
```

**Review by juliaElastic — APPROVED** [review]
LGTM, we could also mention that agent bulk actions are available on more than 10k agents when selecting with `Select everything on all pages` button.

File: docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc [hunk 2]
```diff
@@ -67,6 +70,8 @@ For advanced filtering, use the search bar to create structured queries
 using {kibana-ref}/kuery-query.html[{kib} Query Language]. For example, enter
 `local_metadata.os.family : "darwin"` to see only agents running on macOS.
 
+You can also sort the list of agents by host, last activity time, or version, by clicking on the table headings for those fields. 
+
```
Comment (kilfoyle) [inline docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc:74]:
```suggestion

To perform a bulk action on more than 10,000 agents, you can select the **Select everything on all pages** button.

```

**Comment by kilfoyle** [comment]
Thanks @juliaElastic! I added a sentence about the bulk actions. 

Unfortunately the new commit automatically removed your approval. Would you mind giving it a thumbs up again?

**Pull request merged**